Scripts/Python/plasma/PlasmaTypes.py

```diff
@@ -675,7 +675,7 @@ def __init__(self):
     def getParentKeys(self):
         # if we haven't got the parent keys yet, then add them to the dict
         if not self.gotParentKeys:
-            for anim in dict.values(self):
+            for anim in list(dict.values(self)):
```

Comment:
`list` unnecessary for iteration

Reply:
This has been observed to occur while the dict is changing.